src/SFML/Window/CMakeLists.txt

```diff
@@ -259,7 +265,7 @@ if(SFML_OS_LINUX)
     target_link_libraries(sfml-window PRIVATE UDev)
 elseif(SFML_OS_WINDOWS)
     target_link_libraries(sfml-window PRIVATE winmm gdi32)
-elseif(SFML_OS_FREEBSD)
+elseif(SFML_OS_FREEBSD OR SFML_OS_OPENBSD)
```

Comment:
If we can't interface with joysticks via usbhid do we still need to link to it on OpenBSD?